strapi.json

```diff
@@ -51,24 +51,24 @@
     "SDK [React]": "No",
     "SDK [AngularJS]": "No",
     "SDK [TypeScript]": "No",
-    "Web Hooks": "No",
+    "Web Hooks": "Yes",
     "Eventbus": "No",
     "Bulk Import": "No",
     "Client Side Forms": "Yes",
     "Plugin System": "Yes",
     "Customizable UI": "Yes",
-    "User Management": "",
+    "User Management": "Yes",
     "Role Based Permissions": "Yes",
     "Document Level Permissions": "Yes",
     "OAuth 2.0 Support": "Yes",
     "Auditing": "No",
     "API Keys": "No",
     "Project Support": "No",
     "I18N Localized Content": "Yes",
-    "Content Trees": "No",
+    "Content Trees": "Yes",
```

Comment:
Nesting of fields != Content Trees. IMHO some systems differentiate their data retrial and data management by supporting tree structures. As far as I understand it the content is still perceived flat in Strapi. Does Strapi provide ways to retrieve data via tree path?

Reply:
Oh sorry, I made the change too. My bad. The content isn't flattened but we cannot access it via tree path. I hope it will be possible in the future 🙏